app/locale/en.coffee

```diff
@@ -661,7 +661,7 @@
     unsubscribing: "Unsubscribing"
     subscribe_prepaid: "Click Subscribe to use prepaid code"
     using_prepaid: "Using prepaid code for monthly subscription"
-    feature_levels: "Access __levelsCount__ levels available"
+    feature_levels: "Access __premiumLevelsCount__ levels available"
```

Comment:
I think it would be a good idea to run a script to propagate this change to the other locale files.